.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
Can you kindly remove this from the commit? Also, you may include `.vscode/*` in `.gitignore` so that it won't be committed in the repository in the future :)